Liadi/events-manager
# #152796813 "landing page" template

#### What does this do?
 - Add basic elements to the landing page

#### Task to be completed
 - Page responsiveness
 - Navigation

#### How to test features?
  - Navigate to https://liadi.github.io/events-manager/template/index.html
 - Test responsiveness using developer tools
 OR
 - Clone repo(for commit history)
 - Ensure branch is 'ft-landing-page-152796813' 
 - Open browser and 
 - Navigate to [repo base path]/template/index.html

#### Background context
 - None

#### Related PivotalTracker stories (id)
 - #152833560

#### Screenshots
![landing-page](https://user-images.githubusercontent.com/29649308/32844323-60d87bd4-ca22-11e7-90f0-62c041ca0b72.PNG)
 
#### Questions
 - None
